service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -2,7 +2,8 @@
 
 public enum DocmosisTemplates {
     C6("FL-PLW-HNO-ENG-00092.docx", "Notice_of_proceedings_(c6)"),
-    C6A("FL-PLW-HNO-ENG-00093.docx", "Notice_of_proceedings_(c6a)");
+    C6A("FL-PLW-HNO-ENG-00093.docx", "Notice_of_proceedings_(c6a)"),
+    SDO("FL-PLW-STD-ENG-00099.docx","SDO");
```

Comment:
Super minor, but lack of spaces always annoys me :)

Reply:
I agree here, should SDO -> Standard_directions_order?

Reply:
Probably makes more sense as well, thanks.